packages/fxa-payments-server/src/lib/amplitude.test.ts

```diff
@@ -68,7 +68,11 @@ it('should capture error during logAmplitudeEvent', () => {
 });
 
 it('should call logAmplitudeEvent with subscription plan info', () => {
-  const plan = { plan_id: '123xyz_hourly', product_id: '123xyz' };
+  const plan = {
```

Comment:
Didn't occur to me to nit here last time: since `payment_provider` isn't part of a `plan`, this name is now misleading.